synapse/rest/client/v2_alpha/register.py

```diff
@@ -505,6 +505,7 @@ def __init__(self, hs):
             request,
             body,
             self.hs.get_ip_from_request(request),
+            "log into your account",
```

Comment:
this one isn't really right. Tbh this reflects the fact that UIA is an awkward fit for registration: reg uses it differently to everything else.

This is probably better though:

```suggestion
            "register a new account",
```